spec/providers/social/social.double.integration.src.js

```diff
@@ -37,117 +48,82 @@ module.exports = function(provider_url, setup) {
     });
   }
 
+  function errHandler(err) {
```

Comment:
Sure makes sense. I think the common functions are pretty minimal right now, but if we add more we should think about this certainly